files/SZWESAMP/ZWEIAPF2

```diff
@@ -28,10 +33,15 @@
 //STDPARM DD *
 SH cd '{zowe.runtimeDirectory}' &&
 cd bin/utils &&
-export LOADLIB='{zowe.setup.dataset.authLoadlib}' &&
-export LOADLOC=SMS &&
-export PLUGLIB='{zowe.setup.dataset.authPluginLib}' &&
-export PLUGLOC=SMS &&
+LOADLIB='{zowe.setup.dataset.authLoadlib}' &&
+if [ "$LOADLIB" = "CFG.ZOWE.SETUP.DATASET.AUTHLOADLIB" ]; then
```

Comment:
What's the origin of `CFG.ZOWE.SETUP.DATASET.AUTHLOADLIB` ? Is this generated by configmgr when yaml fields are missing?

Reply:
Yes and no. It is generated by `ZWEGEN00`, technically it is uninitialized `rexx` variable, which is the result of missing yaml property not found by `rexx configmgr`.

Reply:
The question I am asking is: should we make all JCL as simple as possible and do the complicated processing in javascript? Or should we include some logic into JCL to keep the code more simple?

This solution is somewhere in between.

Reply:
Yup, the solution you went with makes sense to me. As a default position I'd support simpler JCL since system programmers and administrators may review it and we'd want to prioritize their experience during install and config, though it's not a hard rule.

I would prefer if `ZWEGEN00` set the `{zowe.setup.dataset.authLoadlib}` variable to something which conveyed an undefined or unset field rather than dumping the name of the rexx variable itself, but its not a huge deal. This looks good as-is.